traits/ctraits.c

```diff
@@ -185,6 +185,9 @@ static int call_notifiers ( PyListObject *, PyListObject *,
    validation. */
 #define MAXIMUM_DEFAULT_VALUE_TYPE 9
 
+/* The maximum value for comparison_mode. Valid values are between 0 and
+   the maximum value. */
+#define MAXIMUM_COMPARISON_MODE_VALUE 3
```

Comment:
Hmm. Shouldn't this be `2` rather than `3`. Else we get a bit of a confusing error message:
```
>>> tr.comparison_mode(3)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: The comparison mode must be 0..3, but 3 was specified.
```

Reply:
Thanks for noticing.  Corrected in 5b0bdc2.